### PR TITLE
[storage] [hotfix] [OCP-18737,OCP-14665,OCP-14673]

### PR DESCRIPTION
**Pull request by ropatil010**
Hi Team, 

PR fix for: https://issues.redhat.com/browse/OCPQE-25056

2 Solutions on TC: OCP-18737, 
1. Add toleration to pods/deployment
2. Deploy the pods on non tainted nodes: Implemented this one currently

Jenkins Logs:
oc get nodes | grep "worker"
NAME                          STATUS   ROLES                  AGE     VERSION
ip-10-0-128-33.ec2.internal   Ready    edge,worker,wscan      5h49m   v1.30.3
ip-10-0-52-209.ec2.internal   Ready    worker,wscan           6h6m    v1.30.3
ip-10-0-79-213.ec2.internal   Ready    worker,wscan           6h7m    v1.30.3
ip-10-0-95-138.ec2.internal   Ready    worker,wscan           6h6m    v1.30.3
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1022559/console
Pod got deployed on node: ip-10-0-52-209.ec2.internal

oc get nodes | grep "worker"
ip-10-0-128-33.ec2.internal   Ready                      edge,worker,wscan      5h50m   v1.30.3
ip-10-0-52-209.ec2.internal   Ready,SchedulingDisabled   worker,wscan           6h7m    v1.30.3
ip-10-0-79-213.ec2.internal   Ready,SchedulingDisabled   worker,wscan           6h8m    v1.30.3
ip-10-0-95-138.ec2.internal   Ready                      worker,wscan           6h7m    v1.30.3
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1022560/
Pod got deployed on node:: ip-10-0-95-138.ec2.internal

/assign @duanwei33 @chao007 @Phaow @radeore

### CODE REVIEW

File: features/step_definitions/node.rb [hunk 1]
```diff
@@ -683,3 +683,8 @@
    admin.cli_exec(:oadm_cordon_node, opts)
   end
 end
+
+Given /^I store the schedulable workers without taints in the :nodes clipboard$/ do
```
Comment (pruan-rht) [inline features/step_definitions/node.rb:687]: @ropatil010 generally looks good...one area of improvement is to make the clipboard name to be a variable as well...not a deal breaker...just a suggestion from my end.

Comment (ropatil010) [reply features/step_definitions/node.rb]: Thanks @pruan-rht. Fixed as per your suggestion.
Logs:  https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1024395/console 

08-29 12:43:36.311  1 scenario (1 passed)

File: features/step_definitions/node.rb [hunk 2]
```diff
@@ -683,3 +683,8 @@
    admin.cli_exec(:oadm_cordon_node, opts)
   end
 end
+
+Given /^I store the schedulable workers without taints in the#{OPT_SYM} clipboard$/ do |cb_name|
+  step %Q{I store the schedulable workers in the :#{cb_name} clipboard}
```
Comment (pruan-rht) [inline features/step_definitions/node.rb:688]: @ropatil010 when using `OPT_SYM`, the variable means the user can put in an optional symbol name, so you need set a default clipboard name is none is defined.  You can do so by  ||
```
cb_name ||= :nodes
```
After doing that, you can call the step like so 
```
I store the schedulable workers without taints in the clipboard
```

Comment (ropatil010) [reply features/step_definitions/node.rb:689]: Fixed, Can you PTAL and help to review. 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1024429/console

File: features/step_definitions/project.rb [hunk 3]
```diff
@@ -193,7 +193,7 @@
     | project_name  | #{project_name}       |
     | node_selector | #{project_name}=label |
     })
-  step %Q/I store the ready and schedulable workers in the :nodes clipboard/
+  step %Q/I store the schedulable workers without taints in the :nodes clipboard/
```
Comment (pruan-rht) [inline features/step_definitions/project.rb:196]: @ropatil010 FYI, now that you have added default value, you can simply call the step like 
```
step %Q/I store the schedulable workers without taints in the clipboard/
```

**Pull request merged**